lstchain/io/event_selection.py

```diff
@@ -89,8 +89,8 @@ class DL3Cuts(Component):
     ).tag(config=True)
 
     fill_theta_cut = Float(
-        help="Fill value of theta cut (deg) in an energy bin with fewer " +
-            "than minimum number of events present",
+        help="Fill value of theta cut (deg) in an energy bin with fewer "
+        + "than minimum number of events present",
         default_value=0.32,
```

Comment:
A fixed default value is probably not the best approach.
I think on the higher energy end we should use the value for the highest energy bin in which the cut could be determined. And for the lower energies, the cut value for the lowest energy bin where it could be determined.

Reply:
But may be something for a separate PR.